toolchain/check/import.cpp

```diff
@@ -216,6 +217,77 @@ static auto CopyEnclosingNameScopesFromImportIR(
   return scope_cursor;
 }
 
+// Returns the canonical IR inst for an entity. Returns an invalid ir_id for the
+// current IR.
+static auto GetCanonicalImportIRInst(Context& context,
+                                     const SemIR::File* cursor_ir,
+                                     SemIR::InstId cursor_inst_id)
+    -> SemIR::ImportIRInst {
+  for (;;) {
```

Comment:
```suggestion
  while (true) {
```
for consistency with how we write this elsewhere.